test/unit/BPool.t.sol

```diff
@@ -69,6 +69,10 @@ abstract contract BasePoolTest is Test, BConst, Utils {
     bPool.set__finalized(_isFinalized);
   }
 
+  function _setBalance(address _user, uint256 _balance, bool _adjustTotalSupply) internal {
```

Comment:
let's not use the boolean and always set it to true, this method should be named so that a reference to being the pool's balance is understood, perhaps `_setPoolBalance`?

in other method we may need `_setPoolTokenBalance` for the erc20 balance of the pool in a token